pkg/controller/build/build_controller_test.go

```diff
@@ -34,6 +33,7 @@ var _ = Describe("Reconcile Build", func() {
 		reconciler                   reconcile.Reconciler
 		request                      reconcile.Request
 		buildSample                  *build.Build
+		secretSample                 *corev1.Secret
```

Comment:
this was very helpful, there was only one issue, will comment on that line, thanks for this!